doc/releases/migration-guide-3.7.rst

```diff
@@ -756,6 +756,21 @@ Bluetooth Host
   longer used in Zephyr 3.4.0 and later. Any references to this field should be removed. No further
   action is needed.
 
+* :code:`BT_LE_ADV_PARAM` now returns a :code:`const` pointer. Any place where the result is stored
```

Comment:
Prob better to use `:c:macro:` here and other relevant places in this PR to help users with cross references